README.md

```diff
@@ -26,6 +26,8 @@ optional arguments:
                         unusual microscope configurations
   --flip-y              flip tile positions top-to-bottom to account for
                         unusual microscope configurations
+  --flip-mosaic-x       flip output image horizontally
```

Comment:
@Yu-AnChen Do you guys consider adding an arg to rotate the output mosaic to 90 degree? I am having this use case right now. I can rotate manually afterwards but it will be great if this can be configured in the command. No rush, just asking. Thank you!

Reply:
Sorry that I missed your comment, we don't have a plan for rotating the mosaics yet. You could do [an editable pip install](https://pip.pypa.io/en/stable/cli/pip_install/#editable-installs) and change the mosaic output logic. 
- https://github.com/labsyspharm/ashlar/blob/2587ea683c924e1b6d3cc4913e17532266471c5d/ashlar/reg.py#L1105-L1108
- https://numpy.org/doc/stable/reference/generated/numpy.rot90.html

And the mosaic shape may need to be changed as well depending on how many 90-degree rotations you've done
- https://github.com/labsyspharm/ashlar/blob/2587ea683c924e1b6d3cc4913e17532266471c5d/ashlar/reg.py#L971-L977